### PR TITLE
Feature/settings usage

### PR DESCRIPTION
**Pull request by iNewLegend**
Manual test:
```
<?php
require __DIR__ . '/../wp-load.php';

$params = [
	'usages' => [],
];

var_export( \Elementor\Modules\Usage\Module::instance()->add_tracking_data( $params ) );
```

### CODE REVIEW

File: modules/usage/module.php [hunk 1]
```diff
@@ -547,6 +548,43 @@ private function get_elements_usage( $elements ) {
...
+	 * Get's the current settings usage.
+	 *
+	 * @return array
+	 */
+	private function get_settings_usage() {
+		$result = [];
+		$settings_tabs = \Elementor\Plugin::$instance->settings->get_tabs();
+
+		foreach ( $settings_tabs as $tab ) {
+			$label = strtolower( $tab['label'] );
```
Comment (matipojo) [inline modules/usage/module.php]: Don't use label, find a key or something else.

File: modules/usage/module.php [hunk 2]
```diff
@@ -547,6 +548,43 @@ private function get_elements_usage( $elements ) {
...
+
+			foreach ( $tab['sections'] as $section ) {
+				if ( ! isset( $section['fields'] ) ) {
+					continue;
+				}
+				foreach ( $section['fields'] as $field_name => $field_values ) {
+					if ( ! strstr( $field_name, 'elementor_' ) ) {
```
Comment (matipojo) [inline modules/usage/module.php]: Always add `elementor_`

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

File: modules/usage/module.php [hunk 3]
```diff
@@ -252,6 +252,7 @@ public function on_before_delete_post( $post_id ) {
 	 */
 	public function add_tracking_data( $params ) {
 		$params['usages']['elements'] = get_option( self::OPTION_NAME );
+		$params['usages']['settings'] = $this->get_settings_usage();
```
Comment (matipojo) [inline modules/usage/module.php]:
```suggestion
		$params['usages']['elementor_settings'] = $this->get_settings_usage();
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — DISMISSED** [review]

**Review by KingYes — CHANGES_REQUESTED** [review]
Use with whitelist && Defaults only.

**Pull request closed without merging**